src/audiomixerboard.cpp

```diff
@@ -1349,29 +1349,37 @@ void CAudioMixerBoard::AutoAdjustAllFaderLevels()
 
             if ( leveldB >= AUTO_FADER_NOISE_THRESHOLD_DB )
             {
-                vecMinLevel[group] = fmin ( vecMinLevel[group], leveldB );
                 vecMaxLevel[group] = fmax ( vecMaxLevel[group], leveldB );
+                levels[group].Add ( leveldB );
             }
             ++vecChannelsPerGroup[group];
         }
     }
 
+    // sort levels for later median computation
```

Comment:
Any `sort` call makes me worry.  There really isn't much data here, however, so I'm going to presume it can never hurt...

Reply:
I know... However, I do not see an alternative for the median, which I believe is the best choice in this case. And median requires sorting.

The worst case would be sorting an array with `MAX_NUM_CHANNELS` elements. Since this is a function which is not called periodically but by the user (CTRL+F or CMD+F), I think this should be acceptable.

By the way, `CAudioMixerBoard::ChangeFaderOrder` also uses sorting ;-)